DigitalLearningSolutions.Data/Models/Centre.cs

```diff
@@ -23,5 +23,9 @@ public class Centre
         public string? OrganisationsCovered { get; set; }
         public string? TrainingVenues { get; set; }
         public string? OtherInformation { get; set; }
+        public int CmsAdministrators { get; set; }
+        public int CmsManagers { get; set; }
+        public int CcLicences { get; set; }
+        public int Trainers { get; set; }

```

Comment:
Should we name these XXXSpots to make it clear that this is what these numbers mean (as I think the db column naming is a bit confusing)